### ISSUE TITLE
Error on API for request states/binary_sensor.snapshots_stale

### ISSUE THREAD

**Issue by PieBru** [sabeechen/hassio-google-drive-backup#96]
Hi Stephen,
I'm using HassIO on Debian Buster Lite with a Raspberry PI4 4GB RAM.

At boot I see these apparently not harmful "error" messages, maybe the startup on the PI is slow and the addon starts too early:
```
19-09-03 08:59:00 INFO (SyncWorker_4) [hassio.docker.homeassistant] Start homeassistant homeassistant/raspberrypi4-homeassistant with version 0.98.2
19-09-03 08:59:03 ERROR (MainThread) [hassio.homeassistant] Error on call http://172.30.32.1:8123/api/states/binary_sensor.snapshots_stale: Cannot connect to host 172.30.32.1:8123 ssl:None [Connection refused]
19-09-03 08:59:03 ERROR (MainThread) [hassio.api.proxy] Error on API for request states/binary_sensor.snapshots_stale
19-09-03 08:59:03 INFO (MainThread) [hassio.api.security] /homeassistant/info access from cebe7a76_hassio_google_drive_backup
19-09-03 08:59:03 INFO (MainThread) [hassio.api.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup
19-09-03 08:59:04 INFO (MainThread) [hassio.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup
19-09-03 08:59:04 INFO (MainThread) [hassio.api.security] /snapshots/aa5b53d4/info access from cebe7a76_hassio_google_drive_backup
19-09-03 08:59:08 INFO (MainThread) [hassio.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup
19-09-03 08:59:28 ERROR (MainThread) [hassio.homeassistant] Error on call http://172.30.32.1:8123/api/states/sensor.snapshot_backup: Cannot connect to host 172.30.32.1:8123 ssl:None [Connection refused]
19-09-03 08:59:28 ERROR (MainThread) [hassio.api.proxy] Error on API for request states/sensor.snapshot_backup
19-09-03 09:00:13 ERROR (MainThread) [hassio.homeassistant] Error on call http://172.30.32.1:8123/api/states/sensor.snapshot_backup: Cannot connect to host 172.30.32.1:8123 ssl:None [Connection refused]
19-09-03 09:00:13 ERROR (MainThread) [hassio.api.proxy] Error on API for request states/sensor.snapshot_backup
19-09-03 09:01:21 INFO (MainThread) [hassio.homeassistant] Detect a running Home Assistant instance
19-09-03 09:01:21 INFO (MainThread) [hassio.addons] Phase 'application' start 4 add-ons
19-09-03 09:01:21 INFO (SyncWorker_1) [hassio.docker.interface] Clean addon_core_configurator application
19-09
```

Thanks,
Piero

**Comment by sabeechen** [comment]
I believe that is just the add-on starting up before hassio finishes booting up the Home Assistant docker container.  The addon should still work fun (but maybe show an error for a few minutes on the Web UI if it tries to create a snapshot at startup).  I'll see if there is a way to make hassio delay the addon startup until HA is running.

**Comment by sabeechen** [comment]
Looks like I can by setting startup: "application" in the [addon config](https://developers.home-assistant.io/docs/en/hassio_addon_config.html#add-on-config).  I'll include this in a minor release soon.

**Comment by sabeechen** [comment]
I just released 0.98.3, which should prevent these errors at startup.  Please re-open the issue if you still see problems.
